lib/utils.coffee

```diff
@@ -293,7 +295,7 @@ exports.requestAsync = (options, retriesRemaining) ->
 	delete opts.timeout
 
 	requestTime = new Date()
-	p = fetch(url, opts)
+	p = exports.fetch(url, opts)
```

Comment:
I'm fairly sure it does. We have to use the exposed mutable reference to fetch, so the tests can change it externally.

I'd love to use a nice mocking framework to mock this without that, but I spent longer than I'd like fighting to set up one that worked happily with both Browserify and Node before Christmas, and I don't think the extra complexity is worth it.